amanuensis/__init__.py

```diff
@@ -100,8 +100,7 @@ def app_register_blueprints(app):
     # google_blueprint = amanuensis.blueprints.google.make_google_blueprint()
     # app.register_blueprint(google_blueprint, url_prefix="/google")
 
-
-    app.register_blueprint(amanuensis.blueprints.search.blueprint, url_prefix="/filter-set")
+    app.register_blueprint(amanuensis.blueprints.filterset.blueprint, url_prefix="/filter-set")
```

Comment:
Shouldn't `url_prefix` value here be `"/filter-sets"` instead?

Reply:
In switching it for testing, I missed that one when I switched it back.  Corrected.